### ISSUE TITLE
train problem

### ISSUE THREAD

**Issue by JayjieL** [pjreddie/darknet#1671]
when i train on my own data, result is nan and nan ....

Even when i follow this :
https://pjreddie.com/darknet/train-cifar/

the result are as follows:
layer     filters    size              input                output
    0 conv    128  3 x 3 / 1    28 x  28 x   3   ->    28 x  28 x 128  0.005 BFLOPs
    1 conv    128  3 x 3 / 1    28 x  28 x 128   ->    28 x  28 x 128  0.231 BFLOPs
    2 conv    128  3 x 3 / 1    28 x  28 x 128   ->    28 x  28 x 128  0.231 BFLOPs
    3 max          2 x 2 / 2    28 x  28 x 128   ->    14 x  14 x 128
    4 dropout       p = 0.50               25088  ->  25088
    5 conv    256  3 x 3 / 1    14 x  14 x 128   ->    14 x  14 x 256  0.116 BFLOPs
    6 conv    256  3 x 3 / 1    14 x  14 x 256   ->    14 x  14 x 256  0.231 BFLOPs
    7 conv    256  3 x 3 / 1    14 x  14 x 256   ->    14 x  14 x 256  0.231 BFLOPs
    8 max          2 x 2 / 2    14 x  14 x 256   ->     7 x   7 x 256
    9 dropout       p = 0.50               12544  ->  12544
   10 conv    512  3 x 3 / 1     7 x   7 x 256   ->     7 x   7 x 512  0.116 BFLOPs
   11 conv    512  3 x 3 / 1     7 x   7 x 512   ->     7 x   7 x 512  0.231 BFLOPs
   12 conv    512  3 x 3 / 1     7 x   7 x 512   ->     7 x   7 x 512  0.231 BFLOPs
   13 dropout       p = 0.50               25088  ->  25088
   14 conv     10  1 x 1 / 1     7 x   7 x 512   ->     7 x   7 x  10  0.001 BFLOPs
   15 avg                        7 x   7 x  10   ->    10
   16 softmax                                          10
Loading weights from backup...Done!
/root/Jayjiel/darknet/data/cifar/test/3661_automobile.png, 1, nan, nan, 
0: top 1: 0.000000, top 2: 1.000000
/root/Jayjiel/darknet/data/cifar/test/4572_bird.png, 2, nan, nan, 
1: top 1: 0.000000, top 2: 0.500000
/root/Jayjiel/darknet/data/cifar/test/416_airplane.png, 0, nan, nan, 
2: top 1: 0.333333, top 2: 0.666667
/root/Jayjiel/darknet/data/cifar/test/4863_automobile.png, 1, nan, nan, 
3: top 1: 0.250000, top 2: 0.750000
/root/Jayjiel/darknet/data/cifar/test/9523_dog.png, 5, nan, nan, 
4: top 1: 0.200000, top 2: 0.600000
/root/Jayjiel/darknet/data/cifar/test/3612_automobile.png, 1, nan, nan, 
5: top 1: 0.166667, top 2: 0.666667
/root/Jayjiel/darknet/data/cifar/test/9090_bird.png, 2, nan, nan, 
6: top 1: 0.142857, top 2: 0.571429
/root/Jayjiel/darknet/data/cifar/test/3443_deer.png, 4, nan, nan, 
7: top 1: 0.125000, top 2: 0.500000
/root/Jayjiel/darknet/data/cifar/test/8274_frog.png, 6, nan, nan, 
8: top 1: 0.111111, top 2: 0.444444
/root/Jayjiel/darknet/data/cifar/test/6245_frog.png, 6, nan, nan, 
9: top 1: 0.100000, top 2: 0.400000
/root/Jayjiel/darknet/data/cifar/test/2356_ship.png, 8, nan, nan, 
10: top 1: 0.090909, top 2: 0.363636
